lib/Ast.ml

```diff
@@ -1,223 +0,0 @@
-(** Abstract syntax. *)
-open Core_kernel
-
-(* for auto generating s-exp *)
-
-(* == Source code locations for semantic errors == *)
-type location =
-  | Location of Lexing.position * Lexing.position  (** delimited location *)
-  | Nowhere  (** no location *)
-
-(* == Unsized types == *)
-type originblock =
-  | Primitives
-  | Functions
-  | Data
-  | TData
-  | Param
-  | TParam
-  | Model
-  | GQuant
-
-and unsizedtype =
-  | Int
-  | Real
-  | Vector
-  | RowVector
-  | Matrix
-  | Array of unsizedtype
-  | Fun of (originblock * unsizedtype) list * returntype
-  | PrimitiveFunction
```

Comment:
MathLibraryFunction but also could you add a comment on why this is in here (that it is used to annotate expressions)

Reply:
Done.